src/fonts/mod.rs

```diff
@@ -228,17 +202,17 @@ mod tests {
     use super::*;
     use crate::{
         drawable::Drawable, geometry::Point, mock_display::MockDisplay, pixelcolor::BinaryColor,
-        style::TextStyle,
+        style::MonoTextStyle,
     };
 
     /// Draws a text using the given font and checks it against the expected pattern.
     pub(super) fn assert_text_from_pattern<F>(text: &str, font: F, pattern: &[&str])
     where
-        F: Font + Copy,
+        F: MonoFont + Copy,
```

Comment:
If you're at it, could you require MonoFont to implement Copy?

Reply:
I think that should be possible, but I'll need to take a closer look.

Reply:
Done.